client/pages/NewEvent.js

```diff
@@ -278,27 +280,28 @@ class NewEvent extends React.Component {
 
     const fromMinutes = getMinutes(fromTime);
     const toMinutes = getMinutes(toTime);
-
+    // create a date range as date
     if (dateOrDay) {
       const dates = [];
 
-      for (const key of Object.keys(weekDays)) {
-        if (!weekDays[key]) continue;
-        dates.push({
-          fromDate: moment()
+      Object.keys(weekDays).forEach((key) => {
```

Comment:
I don't agree. 
- We are using a airBnB rules. So attached and respect the rules. 
- forEach its more readable and enforces immutability.